packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -353,6 +353,12 @@ class AndroidWebViewController extends PlatformWebViewController {
   void Function(ScrollPositionChange scrollPositionChange)?
       _onScrollPositionChangedCallback;
 
+  /// Sets the file access permission for the web view.
+  ///
+  /// Defaults to false.
```

Comment:
This is no longer true. This should probably follow the same language as the [docs](https://developer.android.com/reference/android/webkit/WebSettings#setAllowFileAccess(boolean)): 

```
The default value is true for apps targeting API 29 and below, and false when targeting API 30 and above.
```

Reply:
Good point, I have made the requested change. Thank you